h/jinja_extensions/__init__.py

```diff
@@ -0,0 +1,24 @@
+from urllib.parse import unquote
+
+from h.jinja_extensions import filters
+from h.jinja_extensions.svg_icon import svg_icon
+
+
+def setup_jinja2_env(environment):
+    # 3rd party filters
+    environment.filters["url_unquote"] = unquote
+
+    # 1st party filters
+    environment.filters["to_json"] = filters.to_json
+    environment.filters["human_timestamp"] = filters.human_timestamp
+    environment.filters["format_number"] = filters.format_number
+
+    # 1st party globals
+    environment.globals["svg_icon"] = svg_icon
```

Comment:
We don't really need extensions to achieve what we want, which is just to modify the environment.

Reply:
The `# 3rd party filters` and `# 1st party filters` comments confused me. I thought that `url_unquote` must be coming from a third-party package that we depended on, but it's from the standard library. And I thought the first-party filters must be ones that ship with Jinja itself, but they're our own custom ones. I think we can probably just delete the comments, rather than trying to think of other words for them

Reply:
First party is yourself, third party is other. So 1st party code is code we have written ourselves, and 3rd party code is code we didn't write.

So whether code is in the standard library or a regular library it's still 3rd party code from our point of view. 1st party would only apply to Jinja if we were the developers of Jinja.

https://stackoverflow.com/a/2895801/3569627

I'll swap the comments to use the phrases "Code we've written" / "Code we've imported from someone else" to be more explicit.